### ISSUE TITLE
Nan & Inf loss

### ISSUE THREAD

**Issue by MohammdReza2020** [KunZhou9646/seq2seq-EVC#15]
Hello dear 
thanks for your work.
I am working on your repository.
in pretrain step. at first hundred interations (300, 400), I get Nan or Inf  validation loss.
It is okay or there is a problem?
I attach the screenshot:

![2023-10-13_212735](https://github.com/KunZhou9646/seq2seq-EVC/assets/16300967/d63eb904-81b9-44d7-a310-0d2f4f9bbba3)
